Library/Formula/libbpg.rb

```diff
@@ -1,8 +1,8 @@
 class Libbpg < Formula
   desc "Image format meant to improve on JPEG quality and file size"
   homepage "http://bellard.org/bpg/"
-  url "http://bellard.org/bpg/libbpg-0.9.5.tar.gz"
-  sha256 "30de1d0099920e24b7c9aae4d4e6b62f446823f0a1d52eb195dfc25c662ee203"
+  url "http://bellard.org/bpg/libbpg-0.9.6.tar.gz"
+  sha256 "931abf2744922973e2d57d62b91ef3dd033f0d5d1d829e8f04473f21531311d1"
```

Comment:
This seems wrong? http://bot.brew.sh/job/Homebrew%20Pull%20Requests/37223/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/fetch_libbpg/